frontend/packages/dev-console/integration-tests/features/Helm/helm-Navigation.feature

```diff
@@ -0,0 +1,106 @@
+Feature: Helm Chart
+    User will be able to navigate to Helm tab
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression, @smoke
+Scenario: Open the Helm tab on the navigation bar when helm charts are absent: HR-11-TC02
+    Given user is at the developer perspecitve
+    When user clicks on the Helm tab
+    Then helm releases page will get opened
+    And user will see the message of no helm charts present
+    And user will get the link to install helm charts from developer catalog
+
+
+@regression, @smoke, @ODC-3022, @ODC-2993
+Scenario: Install Helm Chart: HR-02-TC01
+    Given user is at +Add page
+    When user clicks on the Helm Chart card on the +Add page
+    And user searches for the 'Node-ex-k' helm chart
+    And user clicks on the 'Node-ex-k' helm chart card
+    And user clicks on the Install Helm Chart button
+    And user selects the Form View
+    And user enters the required field
+    And user updates the Chart Version
+    And user clicks on the Install button
+    Then user is redirected to Topology page
+    And Topology page have the helm chart workload
+
+
+@regression, @smoke
+Scenario: Open the Helm tab on the navigation bar when helm charts are present: HR-11-TC01
+    Given user is at the developer perspecitve
+    When user clicks on the Helm tab
+    Then helm releases page will get opened
+    And user will see the helm charts listed
+
+
+@regression
+Scenario: Enable the deployed helm charts' filter: HR-11-TC02
+    Given user is at the Helm page
+    When user clicks on the filter drop down menu
+    And user selects checkbox for the Deployed Helm charts
+    Then the checkbox for the Deployed Helm chart is checked
+    And helm charts with status deployed are listed
+
+
+@regression
+Scenario: Enable the failed helm charts' filter: HR-11-TC02
+    Given user is at the Helm page
+    When user clicks on the filter drop down menu
+    And user selects checkbox for the Failed Helm charts
+    Then the checkbox for the Failed Helm chart is checked
+    And helm charts with status failed are listed
+
+
+@regression
+Scenario: Enable the other helm charts' filter: HR-11-TC02
+    Given user is at the Helm page
+    When user clicks on the filter drop down menu
+    And user selects checkbox for the Other Helm charts
+    Then the checkbox for the Other Helm chart is checked
+    And helm charts with status other are listed
+
+
+
+@regression
+Scenario: Select all filters: HR-11-TC02
```

Comment:
Same test case numbers are not allowed for different scenarios 
line 60 and 70 has same test case number. Please update the test cases in dev console sheet, if seperate test cases are required

Reply:
Created new test cases and updated here in the file as well.